etc/mapserver-sample.conf

```diff
@@ -20,6 +20,11 @@ CONFIG
     # MS_DEBUGLEVEL "5"
     # MS_ERRORFILE "/opt/mapserver/logs/mapserver.log"
 
+    #
+    # Global Error Redirect Handling (see web object docs)
+    #
+    # MS_EMPTY "https://mapserver.org/"
```

Comment:
could we call this MS_EMPTY_URL for better expressability ?

Reply:
We certainly could. I agree that's more clear. That could argue for a change to the parameter name in the WEB section for consistency but maybe that's not critical (or a 9.0 change).